payments/models.py

```diff
@@ -31,48 +65,16 @@ def message(self):
         return json.loads(self.message_text)
 
     def save(self, *args, **kwargs):
-        web_hooks[self.type].process(self)
-        super(StripeEvent, self).save(*args, **kwargs)
-
-
-class WebhookRegistry(object):
-
-    def __init__(self):
-        self._registry = {}
-
-    def register(self, webhook):
-        self._registry[webhook.name] = webhook
-
-    def keys(self):
-        return self._registry.keys()
-
-    def __getitem__(self, name):
         try:
-            return self._registry[name]()
+            webhooks[self.type].process(self)
         except KeyError:
-            return Webhook()
-
-web_hooks = WebhookRegistry()
-del WebhookRegistry
-
-
-class Registerable(type):
-    def __new__(cls, clsname, bases, attrs):
-        newclass = super(Registerable, cls).__new__(cls, clsname, bases, attrs)
-        if getattr(newclass, "name", None) is not None:
-            web_hooks.register(newclass)
-        return newclass
-
-
-class Webhook(with_metaclass(Registerable, object)):
-
-    def process(self, event):
-        pass
+            pass
```

Comment:
Need some kind of error handling here ... maybe best to just drop the `except KeyError` so the exception will be thrown up to New Relic.